kettleweb/static/static.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function () {
+                $('#static').on('change', function () {
+                  popCommit();
+                });
+
+                // Because of the way we build forms in wtf this is brittle
+                // so checking the value before making assumptions
+                if ( $('#stages-6').prop('value') == 'deploy_only_prelive_webs') {
+                    $('#stages-6').on('change', function () {
+                        popCommit();
+                    });
+                }
+
+                // populate the short ref in commit box and disable editing
+                // when doing static deploys, don't disable for prelive deploys
+                function popCommit() {
+                    var staticSelect = $.trim($("#static :selected").text()).toLowerCase();
+                    // Again being cautious in case #stages-6 changes
+                    if ( $('#stages-6').prop('value') == 'deploy_only_prelive_webs') {
+                      var preliveWebsDeploy = $('#stages-6').prop('checked')
+                    } else {
+                      var preliveWebDeploy = false
```

Comment:
This one is called preliveWebDeploy and the one above is preliveWeb**S**Deploy is that intentional?